pydevd_frame.py

```diff
@@ -396,6 +399,17 @@ def trace_dispatch(self, frame, event, arg):
                             finally:
                                 if val is not None:
                                     thread.additionalInfo.message = val
+
+                        if event == 'call':
+                            if hasattr(frame, 'f_back'):
+                                back = frame.f_back
+                                if back is not None:
+                                    # When we start debug session, we call execfile in pydevd run function. It produces an additional
+                                    # 'call' event for tracing and we stop on the first line of code twice.
+                                    base = basename(back.f_code.co_filename)
```

Comment:
Use GetFilenameAndBase as it should be faster than basename (as its results are cached)